docs/using.rst

```diff
@@ -722,7 +722,28 @@ Example usage for DNS-01 (Cloudflare API v4) (for example purposes only, do not
        fi
    fi
 
+.. _lock-files:
 
+Lock Files
+==========
+
+When processing a validation Certbot writes a number of lock files on your system
+to prevent multiple instances from overwritting each other's changes. This means
+that two instances of Certbot will not be able to run in parallel.
```

Comment:
nit: You say you can't run Certbot in parallel and then describe how to do so in the final paragraph. Perhaps you should remove this sentence and combine it first sentence with the second paragraph?